cmake/testing.cmake

```diff
@@ -40,13 +40,17 @@ function(f3d_test)
 
   add_test(NAME "${F3D_TEST_NAME}" COMMAND $<TARGET_FILE:f3d> ${F3D_TEST_ARGS} COMMAND_EXPAND_LISTS)
 
-  if(F3D_TEST_LONG_TIMEOUT OR F3D_TEST_INTERACTION)
-    set_tests_properties(${F3D_TEST_NAME} PROPERTIES TIMEOUT 120)
-    if(NOT F3D_ENABLE_LONG_TIMEOUT_TESTS)
-      set_tests_properties(${F3D_TEST_NAME} PROPERTIES DISABLED ON)
-    endif()
+  if(F3D_SANITIZER)
+    set_tests_properties(${F3D_TEST_NAME} PROPERTIES TIMEOUT 3600)
   else()
-    set_tests_properties(${F3D_TEST_NAME} PROPERTIES TIMEOUT 30)
+    if(F3D_TEST_LONG_TIMEOUT OR F3D_TEST_INTERACTION)
+      set_tests_properties(${F3D_TEST_NAME} PROPERTIES TIMEOUT 120)
+      if(NOT F3D_ENABLE_LONG_TIMEOUT_TESTS)
```

Comment:
enabling/disabling long timeout and enabling/disabling sanitizer should both be possible.